scripts/validate.py

```diff
@@ -49,6 +51,29 @@ def validate(src_path, schema_path):
         for item in items:
             try:
                 jsonschema.validate(instance=item, schema=schema)
+                # If it's an experiment we want to evaluate the branches
+                if "arguments" in item:
+                    validated = 0
+                    for branch in item.get("arguments").get("branches"):
+                        if "id" not in branch.get("value"):
+                            print("\tSkip branch", branch.get("slug"))
+                            validated = validated + 1
+                            continue
+                        # Try all of the available message schemas
+                        for try_schema_path in possible_schemas:
+                            with open(try_schema_path, "r") as sf:
+                                message_schema = json.loads(sf.read())
+                                try:
+                                    jsonschema.validate(instance=branch.get("value"), schema=message_schema)
```

Comment:
Nit: looks like we're frequently using `branch.get("value")`, `branch.get("value").get("id")`, and like in this block. Perhaps having some local variables for things `branch.get("value")` would make it easier to read.